### PR TITLE
Added properties

### PR DESCRIPTION
**Pull request by jtodevs**
Added property for field reference in position.
Add elements for Packet

### CODE REVIEW

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Hello @jtodevs, thanks for your interest in czml3! I would be happy to add all these remaining properties to `Packet`, but we would need to work a bit more.

For example, [the `Corridor` property](https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Corridor) defines its own set of properties, so we would need to create it before being able to add it to a `Packet`. Same goes for all the rest.

On the other hand, some Continuous Integration checks are failing, in particular the syntax, see https://circleci.com/gh/poliastro/czml3/533. You can fix that locally by running

```
$ tox -e reformat
```

Let me know if I can be of any help here, or consider [joining our chat](https://riot.im/app/#/room/#poliastro-czml:matrix.org).

**Comment by astrojuanlu** [comment]
While I review this: would you like to add your email to https://github.com/settings/emails so your commits are properly attributed to your account at @jtodevs?

File: src/czml3/properties.py [hunk 1]
```diff
@@ -396,6 +471,34 @@ class Point(BaseCZMLObject):
...
+    maximumScreenSpaceError = attr.ib(default=None)
+
+
+@attr.s(repr=False, frozen=True, kw_only=True)
+class Wall(BaseCZMLObject):
+    """A two-dimensional wall defined as a line strip and optional maximum and minimum heights.
+     It  conforms to the curvature of the globe and can be placed along the surface or at altitude."""
+
+    show = attr.ib(default=None)
+    positions = attr.ib(default=None)
```
Comment (astrojuanlu) [inline src/czml3/properties.py]: This appears to be required: https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Wall#properties

Comment (astrojuanlu) [reply src/czml3/properties.py]:
```suggestion
    positions = attr.ib()
```

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Just left one tiny suggestion, otherwise this looks great and I will merge in the next round when the tests pass.

File: src/czml3/properties.py (hunk 1 above)
Comment (jtodevs) [reply src/czml3/properties.py]: Yes indeed!

**Review by astrojuanlu — APPROVED** [review]
:rocket:

**Comment by astrojuanlu** [comment]
Thanks again @jtodevs, merged!

**Pull request merged**